test/passenger_test.rb

```diff
@@ -69,6 +79,78 @@
   end
 
   describe "net_expenditures" do
-    # You add tests for the net_expenditures method
```

Comment:
`net_expenditures` should explicitly test the edge case of a driver that has trips in progress. You've done a great job addressing this in your methods.. make sure to test the functionality.